compiler/optimizer/VPHandlers.cpp

```diff
@@ -12196,9 +12200,7 @@ TR::Node *constrainArrayStoreChk(OMR::ValuePropagation *vp, TR::Node *node)
    TR::Node *child = node->getFirstChild();
    TR::Node *objectRef;
    TR::Node *arrayRef;
-   bool canBeRemoved = false;
    bool mustFail = false;
-   bool isGlobal;
 
    objectRef = child->getSecondChild();
    arrayRef = child->getChild(2);
```

Comment:
Maybe these lines could be cleaned up 
```suggestion
   TR::Node *objectRef = child->getSecondChild();
   TR::Node *arrayRef = child->getChild(2);
   bool mustFail = false;
```

Reply:
Cleaned this up in commit e516f9868d11f2483bd2f917ede6b879a1271006